app/models/organization.rb

```diff
@@ -38,6 +38,8 @@ class Organization < ActiveRecord::Base
 
   default_scope { order('name asc') }
 
+  scope :only_categories, lambda {|category_name_array| joins(:organization_category).where(organization_categories: {name: category_name_array}) }
+  scope :by_not_category, lambda {|category_name| joins(:organization_category).where.not(organization_categories: {name: category_name}) }
```

Comment:
Remove the :by_not_category scope. There is no need for unused code and it's a simple fix.